### ISSUE TITLE
Can User model be parametrized?

### ISSUE THREAD

**Issue by jrullan** [mdmsoft/yii2-admin#285]
Hello, 

I love the Yii2-Admin module, but the fact that the identity class (User model) is part of the module, limits my implementations which use an application specific User class in `app\models\User.php`.
It would be great if I could map the User class used for Identity to any other class for example an `app\models\User` class that inherits from the `admin\models\User` class.  Then I could tell the module that instead of using the `admin\models\User` class it uses my custom implementation of User.php located at `app\models\User.php`.

Is this supported in version 2.0?

**Comment by mdmunir** [comment]
> but the fact that the identity class (User model) is part of the module.

Nope. feel free use your own User model. https://github.com/mdmsoft/yii2-admin/blob/master/docs/guide/basic-usage.md#admin-module

**Comment by napravicukod** [comment]
It would be good to apply the parameterization to the UserSearch model in UserController as well (searchClass parameter).

**Comment by mdmunir** [comment]
@napravicukod Feel free to submit pull request.
